_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw.cpp

```diff
@@ -848,7 +848,7 @@ mfxStatus   MFXVideoENCODEH265_HW::Reset(mfxVideoParam *par)
         return MFX_ERR_INCOMPATIBLE_VIDEO_PARAM;
 
     // waiting for submitted in driver tasks
-    MFX_CHECK_STS(WaitingForAsyncTasks(isIdrRequired));
+    MFX_CHECK_STS(WaitingForAsyncTasks(isIdrRequired, false));
```

Comment:
Please, pull it off the ```MFX_CHECK_STS``` macro, it will cause double call on unhappy path. I suggest to use ```MFX_SAFE_CALL``` instead

Reply:
> Please, pull it off the `MFX_CHECK_STS` macro, it will cause double call on unhappy path. I suggest to use `MFX_SAFE_CALL` instead

Changed.